connectivity/FEATURE_BLE/source/generic/SecurityManagerImpl.cpp

```diff
@@ -302,9 +346,14 @@ ble_error_t SecurityManager::requestPairing(connection_handle_t connection)
 #if BLE_ROLE_PERIPHERAL
 ble_error_t SecurityManager::acceptPairingRequest(connection_handle_t connection)
 {
-    if (!_db) return BLE_ERROR_INITIALIZATION_INCOMPLETE;
+    tr_info("Connection %d - Accept pairing request", connection);
+    if (!_db) {
+        tr_error("Failure, db not initialized");
```

Comment:
So it can be collapsed with the rest of them
```suggestion
        tr_error("Failure, DB not initialized");
```

Reply:
I guess same below, some copy pasting would solve this to avoid accidental differences in lines.